From 67fcf66904c7f1f6da858eba35e95dad670427c0 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Wed, 17 Jan 2024 14:16:10 +0000
Subject: [PATCH] docs: Remove paragraph about -ip passwords and ssh/scp

The documentation suggested (correctly) that the -ip option did not
cover all cases where we make an ssh connection to either VMware or
Xen.  However with the recent change where we use nbdkit-ssh-plugin in
preference to ssh/scp (commit fb72e05986, "virt-v2v: -i vmx: Replace
external ssh/scp with nbdkit-ssh-plugin") this should no longer be
true for VMware.  And I don't think it was true for Xen for a while.

Therefore remove this paragraph in the documentation (in two places).

See-also: https://bugzilla.redhat.com/show_bug.cgi?id=1854275
See-also: https://lists.libguestfs.org/archives/list/guestfs@lists.libguestfs.org/thread/CKBEYJHUZWLDUUDWAJM2JU6QIKZGTD6F/
Suggested-by: Laszlo Ersek
---
 docs/virt-v2v-input-vmware.pod | 5 -----
 docs/virt-v2v-input-xen.pod    | 5 -----
 2 files changed, 10 deletions(-)

diff --git a/docs/virt-v2v-input-vmware.pod b/docs/virt-v2v-input-vmware.pod
index 48849f88..fe59222a 100644
--- a/docs/virt-v2v-input-vmware.pod
+++ b/docs/virt-v2v-input-vmware.pod
@@ -155,11 +155,6 @@ virt-v2v server to the ESXi hypervisor.  For example:
  $ ssh root@esxi.example.com
  [ logs straight into the shell, no password is requested ]
 
-Note that support for non-interactive authentication via the I<-ip>
-option is incomplete.  Some operations remain that still require the
-user to enter the password manually.  Therefore ssh-agent is recommended
-over the I<-ip> option.  See L<https://bugzilla.redhat.com/1854275>.
-
 =head3 VMX: Construct the SSH URI
 
 When using the SSH input transport you must specify a remote
diff --git a/docs/virt-v2v-input-xen.pod b/docs/virt-v2v-input-xen.pod
index 4a0544f8..38dc8f7b 100644
--- a/docs/virt-v2v-input-xen.pod
+++ b/docs/virt-v2v-input-xen.pod
@@ -32,11 +32,6 @@ server to the Xen host.  For example:
  $ ssh root@xen.example.com
  [ logs straight into the shell, no password is requested ]
 
-Note that support for non-interactive authentication via the I<-ip>
-option is incomplete.  Some operations remain that still require the
-user to enter the password manually.  Therefore ssh-agent is recommended
-over the I<-ip> option.  See L<https://bugzilla.redhat.com/1854275>.
-
 With some modern ssh implementations, legacy crypto algorithms required
 to interoperate with RHEL 5 sshd are disabled.  To enable them, you may
 need to add the following C<Host> stanza to your F<~/.ssh/config>:
